cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -42,22 +47,28 @@ $ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformSt
 $ export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 $ export SCRATCH="/tmp/${CLUSTER_NAME}/dynamic-certs"
 $ mkdir -p ${SCRATCH}
-$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
 ----
 <1> The custom domain.
 <2> The e-mail Let's Encrypt will use to send notifications about your certificates.
+. Ensure all fields output correctly before moving to the next section:
++
+[source,terminal]
+----
+$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
```

Comment:
This echo command doesn't contain all of the fields we just defined - @kmcolli Should we be checking all fields here?

Reply:
@kmcolli @michaelryanmcneill Can you please advise on this? Many thanks.

Reply:
@kmcolli and @michaelryanmcneill Bumping again to get clarity on this. Many thanks.

Reply:
I don't believe it is necessary to echo all content out but we can if you feel it is important.